addons/web/static/src/views/layout.js

```diff
@@ -2,38 +2,23 @@
 
```

Comment:
should we move this file in another folder since it is not limited to views anymore?